packages/create-turbo/src/transforms/official-starter.ts

```diff
@@ -20,7 +20,7 @@ export async function transform(args: TransformInput): TransformResult {
   const defaultExample = isDefaultExample(example.name);
   const isOfficialStarter =
     !example.repo ||
-    (example.repo.username === "vercel" && example.repo.name === "turbo");
+    (example.repo.username === "vercel" && example.repo.name === "turborepo");
```

Comment:
Let's support both turbo and turborepo, since they will both redirect to the right place and work!

Reply:
:nods: Good catch.